rosdep/python.yaml

```diff
@@ -3452,6 +3452,14 @@ python-scipy:
     yakkety_python3: [python3-scipy]
     zesty: [python-scipy]
     zesty_python3: [python3-scipy]
+python-scp:
+  debian: [python-scp]
+  fedora: [python-scp]
+  ubuntu: [python-scp]
+python3-scp:
```

Comment:
Thanks for iterating @Vuwij. Since we keep keys sorted by lexical order, the python3 keys end up in another section of the file. Could you move them there?

Reply:
@Vuwij Please update this to be in alphabetical order so it will pass the linters, then we can merge it.